deltachat-ios/Helper/Utils.swift

```diff
@@ -165,6 +165,11 @@ struct Utils {
     }
 
     public static func authenticateDeviceOwner(reason: String, callback: @escaping () -> Void) {
+
+#if DEBUG
+        return callback()
+#endif
+
```

Comment:
let's get that out, it is unrelated and there is also quite some worth in devs using the same dogfood as the normal users, esp. for some potentially breaking stuff, it is good to have that tested early and by the devs